vars/parseStageInfo.groovy

```diff
@@ -36,6 +36,33 @@
  *                             Default determined by this function below.
  */
 
+String get_commit_pragma_tags(String pragma_suffix) {
+  // Get the test tags defined in the commit message with the following priority:
+  //  1) Test-tag-<stage>: <tags>
+  //  2) Test-tag: <tags>
+  //  3) Features: <feature_tags>
+  String pragma_tag
+  pragma_tag = commitPragma("Test-tag" + pragma_suffix, null)
+  if (!pragma_tag) {
+    pragma_tag = commitPragma("Test-tag", null)
+    if (!pragma_tag) {
+      pragma_tag = commitPragma("Features", null)
+      if (pragma_tag) {
+        String features = pragma_tag
+        pragma_tag = 'pr '
+        for (feature in features.split(' ')) {
+          pragma_tag += 'daily_regression,' + feature + ' '
+          /* DAOS-6468 Ideally we'd like to add this but there are too
+                      many failures in the full_regression set 
+          pragma_tag += 'full_regression,' + feature + ' '
+          */
+        }
+      }
+    }
+  }
+  return pragma_tag
```

Comment:
Now that this evaluation is out on it's own, it might read and be understood a bit better if it wasn't nested so deeply:
```suggestion
  pragma_tag = commitPragma("Test-tag" + pragma_suffix, null)
  if (pragma_tag) {
    return pragma_tag
  }

  pragma_tag = commitPragma("Test-tag", null)
  if (pragma_tag) {
    return pragma_tag
  }
  
  pragma_tag = commitPragma("Features", null)
  if (pragma_tag) {
    String features = pragma_tag
    pragma_tag = 'pr '
    for (feature in features.split(' ')) {
      pragma_tag += 'daily_regression,' + feature + ' '
      /* DAOS-6468 Ideally we'd like to add this but there are too
                  many failures in the full_regression set 
      pragma_tag += 'full_regression,' + feature + ' '
      */
    }
  }
  return pragma_tag
```